app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
After reading the thread, I'm OK with this `can :edit, ::SolrDocument` solution for now. Because we're using `::SolrDocument` and not `:all`, this steers clear of what @cbeer was warning us against. Inclined to merge this work as soon as @awead can get Travis :green_apple:.